### ISSUE TITLE
Segmentation fault on Fedora KDE Wayland

### ISSUE THREAD

**Issue by CDotNightHawk** [Vector35/binaryninja-api#5401]
**Version and Platform (required):**
- Binary Ninja Version: 4.0.4958 (Stable)
- OS: Fedora
- OS Version: 40
- DE Version: KDE Plasma 6.0.4
- WM: KWin (Wayland)
- CPU Architecture: x86_64

**Bug Description:**
Segmentation fault on opening Binary Ninja

**Steps To Reproduce:**
Please provide all steps required to reproduce the behavior:
1. Download latest version
2. Run the program and input license
3. Program closer

**Expected Behavior:**
The main interface of Binary Ninja should open without any error screens.

**Screenshots:**
[![asciicast](https://asciinema.org/a/du55lHjpITYzRPArLpzk05I9G.svg)](https://asciinema.org/a/du55lHjpITYzRPArLpzk05I9G)

**Additional Information:**
On deleting the license.dat file and reproducing the error, I managed to get a stack trace.
```
[0:140157228242176 Default error] Exception ignored on calling ctypes callback function: <bound method ScriptingProvider._create_instance of <binaryninja.scriptingprovider.PythonScriptingProvider object at 0x7f78c8375190>>
[0:140157228242176 Default error] Traceback (most recent call last):
[0:140157228242176 Default error]   File "/home/nighthawk/binaryninja/plugins/../python/binaryninja/scriptingprovider.py", line 418, in _create_instance
[0:140157228242176 Default error]     def _create_instance(self, ctxt):
[0:140157228242176 Default error]    
[0:140157228242176 Default error] KeyboardInterrupt: 
[0:140157228242176 Default error] Exception ignored on calling ctypes callback function: <bound method PythonScriptingProvider._install_modules of <binaryninja.scriptingprovider.PythonScriptingProvider object at 0x7f78c8375190>>
[0:140157228242176 Default error] Traceback (most recent call last):
[0:140157228242176 Default error]   File "/home/nighthawk/binaryninja/plugins/../python/binaryninja/scriptingprovider.py", line 1407, in _install_modules
[0:140157228242176 Default error]     modules = _modules.decode("utf-8")
[0:140157228242176 Default error]               ^^^^^^^^^^^^^^^^^^^^^^^^
[0:140157228242176 Default error] UnicodeDecodeError: 'utf-8' codec can't decode byte 0xd0 in position 0: invalid continuation byte
[0:140157228242176 Default error] Exception ignored on calling ctypes callback function: <bound method PythonScriptingProvider._install_modules of <binaryninja.scriptingprovider.PythonScriptingProvider object at 0x7f78c8375190>>
[0:140157228242176 Default error] Traceback (most recent call last):
[0:140157228242176 Default error]   File "/home/nighthawk/binaryninja/plugins/../python/binaryninja/scriptingprovider.py", line 1407, in _install_modules
[0:140157228242176 Default error]     modules = _modules.decode("utf-8")
[0:140157228242176 Default error]               ^^^^^^^^^^^^^^^^^^^^^^^^
[0:140157228242176 Default error] UnicodeDecodeError: 'utf-8' codec can't decode byte 0x8a in position 2: invalid start byte
Terminating with uncaught exception
Here's a stack trace (hopefully):
/lib64/libstdc++.so.6(+0xb7b7c) [0x7f78e52b7b7c]
/lib64/libstdc++.so.6(_ZSt10unexpectedv+0) [0x7f78e52a5941]
/lib64/libstdc++.so.6(+0xb7e08) [0x7f78e52b7e08]
/lib64/libstdc++.so.6(_ZSt20__throw_system_errori+0x83) [0x7f78e52a9309]
/home/nighthawk/binaryninja/libbinaryninjacore.so.1(BNRegisterScriptingInstanceOutputListener+0xe4) [0x7f78e80c8214]
/home/nighthawk/binaryninja/libbinaryninjaui.so.1(_ZN16ScriptingConsoleC2EP7QWidgetRK7QStringS4_N11BinaryNinja3RefINS5_17ScriptingInstanceEEE+0x9ac) [0x7f78f23400ec]
./binaryninja(+0x16db9a) [0x556f26679b9a]
./binaryninja(+0x179a14) [0x556f26685a14]
./binaryninja(+0x1878c3) [0x556f266938c3]
./binaryninja(+0xdd99f) [0x556f265e999f]
/lib64/libc.so.6(+0x2a088) [0x7f78e503d088]
/lib64/libc.so.6(__libc_start_main+0x8b) [0x7f78e503d14b]
./binaryninja(+0xe92ce) [0x556f265f52ce]

Aborted (core dumped)
```

**Comment by psifertex** [comment]
Duplicate of #5394 and #5392 

Leaving open this time so it will be more discoverable to prevent duplicates but the short explanation is this is an upstream bug and in an unsupported operating system.

**Comment by CDotNightHawk** [comment]
Sorry on the duplicate, thank you for the response to the other listings.

**Comment by D0ntPanic** [comment]
Fixed in 4.1.5307
